man/man8/zed.8.in

```diff
@@ -1,6 +1,5 @@
 .\"
-.\" This file is part of the ZFS Event Daemon (ZED)
-.\" for ZFS on Linux (ZoL) <https://zfsonlinux.org/>.
+.\" This file is part of the OpenZFS Event Daemon (ZED)
```

Comment:
```suggestion
.\" This file is part of the ZFS Event Daemon (ZED)
```

Reply:
Agreed, ZED is not an anagram for OpenZFS Event Daemon...

Reply:
Updated.